autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/obstacle_cruise_planner.param.yaml

```
@@ -179,10 +179,13 @@
 
     slow_down:
       # parameters to calculate slow down velocity by linear interpolation
-      min_lat_margin: 0.2
-      max_lat_margin: 1.0
-      min_ego_velocity: 2.0
-      max_ego_velocity: 8.0
+      labels:
+        - "default"
+      default:
+        min_lat_margin: 0.2
+        max_lat_margin: 1.0
+        min_ego_velocity: 2.0
+        max_ego_velocity: 8.0
```

better to set same to universe?
```suggestion
      labels:
        - "default"
        - "pedestrian"
      default:
        min_lat_margin: 0.2
        max_lat_margin: 1.0
        min_ego_velocity: 2.0
        max_ego_velocity: 8.0
      pedestrian:
        min_lat_margin: 0.5
        max_lat_margin: 1.5
        min_ego_velocity: 1.0
        max_ego_velocity: 2.0
```

---

I left the pedestrian changes in universe to serve as a guide on how to implement custom parameters, but in launcher I don't really want to change the behavior of the vehicle, so I only left the default parameters as they were before my changes so that is why they are different. Thanks for asking and noticing!